debezium-core/src/main/java/io/debezium/converters/CloudEventsMaker.java

```diff
@@ -91,6 +91,12 @@ public static CloudEventsMaker create(RecordParser parser, SerializerType conten
                 return new MongodbCloudEventsMaker(parser, contentType, dataSchemaUriBase);
             case "sqlserver":
                 return new SqlserverCloudEventsMaker(parser, contentType, dataSchemaUriBase);
+            case "oracle":
+                return new OracleCloudEventsMaker(parser, contentType, dataSchemaUriBase);
+            case "db2":
+                return new Db2CloudEventsMaker(parser, contentType, dataSchemaUriBase);
+            case "vitess":
+                return new VitessCloudEventsMaker(parser, contentType, dataSchemaUriBase);
```

Comment:
As I'm seeing this, this had been on my list of things to get rid of for a long time. This is really a poor design, requiring this sort of centralized registry. The idea to overcome is was to use the service loader mechanism, so that each connector can have its own local `CloudEventsMaker` and announce it via some service contract. Perhaps you could look into this while making the changes here anyways?

Reply:
Added a new `CloudEventsProvider` service contract and moved all parser/maker implementations to their respective connector modules.